client/src/main/scala/drt/client/components/DashboardTerminalSummary.scala

```diff
@@ -67,8 +67,8 @@ object DashboardTerminalSummary {
 
   def groupFlightsByHour(flights: List[ApiFlightWithSplits], startMin: SDateLike): Seq[(MillisSinceEpoch, Set[ApiFlightWithSplits])] = {
     val hourInMillis = 3600000
-    flights.sortBy(_.apiFlight.PcpTime).groupBy(fws => {
-      val hoursSinceStart = ((fws.apiFlight.PcpTime - startMin.millisSinceEpoch) / hourInMillis).toInt
+    flights.sortBy(_.apiFlight.PcpTime.getOrElse(0L)).groupBy(fws => {
```

Comment:
we should never have any arrivals that don't have PCP time here, so this Else will never be reached, but I think it's slightly misleading to have this code here, so perhaps we should do a collect first, and weed out any Nones for PCP time before sorting....